feature-libs/product-configurator/rulebased/components/attribute/types/multi-selection-image/configurator-attribute-multi-selection-image.component.ts

```diff
@@ -102,4 +104,8 @@ export class ConfiguratorAttributeMultiSelectionImageComponent
       isLightedUp: value.selected,
     };
   }
+
+  getAriaLabel(attribute: Configurator.Attribute, value: Configurator.Value) {
```

Comment:
There is no need to define this method. 
`getAriaLabelGeneric` could be called directly in HTML tree.
See `createAttributeUiKey` in `configurator-attribute-header.component.html`